tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -374,6 +375,18 @@ func NewBalanceWitnessSchedulerCommand() *cobra.Command {
 	return c
 }
 
+// NewBalanceRangeSchedulerCommand returns a command to add a balance-range-scheduler.
+func NewBalanceRangeSchedulerCommand() *cobra.Command {
+	c := &cobra.Command{
+		Use:        "balance-range-scheduler [--format=raw|encode|hex] <engine> <role> <table-name> <start_key> <end_key>",
+		Short:      "add a scheduler to balance region for given range",
+		Run:        addSchedulerForBalanceRangeCommandFunc,
+		Deprecated: "balance-range will be deprecated in the future, please use sql instead",
```

Comment:
It's strange to add `Deprecated` now.

Reply:
removed and add one todo for sql implement